content/javascript/concepts/arrays/terms/findLastIndex/findLastIndex.md

```diff
@@ -0,0 +1,77 @@
+---
+Title: '.findLastIndex()'
+Description: 'Iterates through the array in reverse order and returns the index that passes the provided testing function. If no item passes, returns -1.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Arrays'
+  - 'Methods'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `.findLastIndex()` iterates through the array in reverse order and returns the index that passes the provided testing function. If no item passes, returns -1.
+
+## Syntax
+
+```pseudo 
+array.findLastIndex((element, index) => {...});
+```
+
+The function can be invoked with two arguments:
+
+- `element`: The current element we are iterating through.
+- `index` (optional): The index of the array element.
+
+## Example
+
+Find the index with a value of 200 in the array:
+
+```js
+const numbers = [10, 42, 53, 12, 45, 66, 33, 6, 10000, 200, 10];
+
+const findIndexResult = numbers.findLastIndex((value, index) => {
+  return value === 200;
+});
+
+console.log(findIndexResult);
+```
+
+This results in the following output:
+
+```shell
+9
+```
+
+## Codebyte Example
```

Comment:
Unfortunately the `findLastIndex()` function isn't recognized by codebytes. So we can't have a codebyte for this entry. However, I think this is a good example, so could you revise this to be "Example 2" with the same code in a ` ```js ` block, and include an output block afterwards?

Reply:
Resolved with commit ea86dbb8803f93d8e57390b0045cd3c3f18ded5c